packages/node/src/edge-functions/edge-handler.ts

```diff
@@ -192,6 +192,11 @@ function extractUrlPath(entrypointRelativePath: string) {
   if (parts.length === 1) {
     return entrypointRelativePath;
```

Comment:
This early return isnt account for. How about this:

```js
const parts = entrypointRelativePath.split('.');
  if (parts.length > 1) {
    parts.pop();
  }
  return `/${parts.join('.')}`;
```

Reply:
definitely going too fast here, yeah, I'll update this

Reply:
@styfle @cb1kenobi or should I use `entrypointToOutputPath`? It doesn't return the `/` at the start, but this function does already essentially exist.

Reply:
I do prefer the implementation of `entrypointToOutputPath()` better, I'm concerned about the potential breakage from touching something shared. In theory the tests would catch it, so I'd say give it a shot. Add `/` to the result of `entrypointToOutputPath()` and replace instances of `extractUrlPath()` with `entrypointToOutputPath()`.

Reply:
How about this?

```ts
function entrypointToRequestPath(entrypoint: string, zeroConfig?: boolean) {
  return `/${entrypointToOutputPath(entrypoint, zeroConfig)}`;
}
```

Reply:
How about this? [`0e8d8b1` (#9428)](https://github.com/vercel/vercel/pull/9428/commits/0e8d8b1694c1762cb9dbab9bbb9dfa30109fb2a9)